controller/sql-query-controller.js

```diff
@@ -0,0 +1,9 @@
+const model = require("../model/sql-query-model.js");
+
+exports.getAllTopics = async function(req, res)
+{
+    const allTopics = await model.queryAllTopics();
+    
+    try { res.status(200).send({"topics": allTopics}) } 
+    catch (error) { console.error(error); }
+}
```

Comment:
Really nice setup of the controller, Seb, and good implementation of the try catch syntax for async/await.